Game/model/model.gd

```diff
@@ -16,10 +16,15 @@ var player_ids: Array[int]
 @onready var player_states: PlayerStates  = %PlayerStates
 @onready var player_spawner := %PlayerSpawner
 @onready var game_state := %GameState
+@onready var _update_timer := %UpdateTimer
 
 
 func start_game() -> void:
 	player_states.start_game()
+	
+	if multiplayer.is_server():
+		# Start the timer on the server and only on the server.
+		_update_timer.start()
```

Comment:
future implementation detail: timer starts when first building is placed. this is good for now.